src/components/ProjectTypeSelection/ProjectTypeSelection.js

```diff
@@ -0,0 +1,75 @@
+import React, { PureComponent } from 'react';
+import styled from 'styled-components';
+
+import reactIconSrc from '../../assets/images/react-icon.svg';
+import gatsbyIconSrc from '../../assets/images/gatsby_small.png';
+
+import FormField from '../FormField';
+import ButtonWithIcon from '../ButtonWithIcon';
+import Spacer from '../Spacer';
+import FadeIn from '../FadeIn';
+
+import { ProjectType } from '../../types';
+type Props = {
+  activeField: string,
+  label?: string,
+  projectType: string,
+  onSelect: string => void,
+};
+
+class ProjectTypeSelection extends PureComponent<Props> {
+  static defaultProps = {
+    label: 'Project Type',
+  };
+
+  select = (ev: SyntheticEvent<*>, projectType: ProjectType) => {
+    ev.preventDefault();
+    this.props.onSelect(projectType);
+  };
+
+  render() {
+    const { activeField, projectType, label } = this.props;
+    return (
+      <FadeIn>
+        <FormField label={label} isFocused={activeField === 'projectType'}>
+          <ProjectTypeTogglesWrapper>
+            <ButtonWithIcon

```

Comment:
instead of copy-pasting all this code, it would have been better to abstract it in a component and use it both the MainPanel and in the AppSettingModal.

As a result, this is now missing the Next.js project type

Reply:
Ah OK, I missed that. I'll add this later today. 

`ProjectTypeSelection` should be used in MainPanel as well.
I'll add Next.js type to `ProjectTypeSelection` and use it in MainPanel.

Reply:
I think Next.js is also missing from `SummaryPane.js`